content/guards.md

```diff
@@ -142,6 +142,8 @@ export class AppModule {}
 > where the guard (`RolesGuard` in the example above) is defined. Also, `useClass` is not the only way of dealing with
 > custom provider registration. Learn more [here](/fundamentals/custom-providers).
 
+> warning **Warning** When using the provider syntax with `APP_GUARD`, it is not necessary to use the `useGlobalGuards()` method of the Nest application instance in the `main.ts` file.
```

Comment:
instead of adding this warning (that sounds more like an _info_), what about changing the above sentence

> In order to solve this issue, you can set up a guard directly from any module using the following construction:

to something that clearify indicates that we should replace `app.useGlobalGuards()` with the `APP_GUARD` provider? :thinking:

Reply:
It could also go, I preferred to add a warning so as not to touch the already documented part that seems very good to me.

Reply:
I think this should be clear & obvious. No need to add an extra warning for this